src/communication/stream_transfer_receiver.cpp

```diff
@@ -47,7 +47,7 @@ namespace cubstream
       void execute () override
       {
 	css_error_code rc = NO_ERRORS;
-	std::size_t max_len = cubcomm::MTU;
+	std::size_t max_len = DB_ALIGN (cubcomm::MTU, MAX_ALIGNMENT);
```

Comment:
I think assert is enough here.

Reply:
cubcomm::MTU was also change to be aligned, but that is not required for a generic communication channel. 
The alignment to 8 bytes is required only by replication stream.
Even this code is "cubstream"  - more generic than replication stream, but there is no simpler way to enforce alignment only for replication stream (we would need max_len as argument to stream_transfer_receiver constructor).

Reply:
I'd like to propose to use `DB_ALIGN_BELOW`.